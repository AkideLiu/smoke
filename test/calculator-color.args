--color fixtures/calculator fixtures/calculator.rb
