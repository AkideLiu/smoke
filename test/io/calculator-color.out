addition
[32m  succeeded[0m
subtraction
[32m  succeeded[0m
multiplication
[32m  succeeded[0m
division
[32m  succeeded[0m
three-numbers
[31m  input:  [0m5 + 7 + 12
[31m  output: [0m[36m@@ -1 +1 @@[m
          [31m-24[m
          [32m+[m[32m12[m
parentheses
[31m  input:  [0m(6 + 3) * 2
[31m  output: [0m[36m@@ -1 +1 @@[m
          [31m-18[m
          [32m+[m[32m3[m
complex
[31m  input:  [0m(8 + 2i) + (7 - 4i)
[31m  output: [0m[36m@@ -1 +1 @@[m
          [31m-15 - 2i[m
          [32m+[m[32m2[m
infix-words-error
[32m  succeeded[0m

[31m8 tests, 3 failures[0m
